package.json

```diff
@@ -31,6 +31,13 @@
   "main": "./dist/lib/server-impl.js",
   "scripts": {
     "start": "TZ=UTC node ./dist/server.js",
+    "build:backend": "copyfiles -u 1 src/mailtemplates/**/*.mustache dist/ && tsc --pretty --strictNullChecks false",
+    "build:frontend": "cd ./frontend && yarn run build",
+    "build": "yarn run build:frontend && yarn run build:backend",
```

Comment:
Yes! With @sjaanus we noticed we'll need `build` to build both backend and frontend :+1: